stage1/init/init.go

```diff
@@ -279,7 +279,7 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, canMachinedRegister boo
 		// kernel and hypervisor binaries are located relative to the working directory
 		// of init (/var/lib/rkt/..../uuid)
 		// TODO: move to path.go
-		kernelPath := filepath.Join(common.Stage1RootfsPath(p.Root), "bzImage")
+		kernelPath := filepath.Join(common.Stage1RootfsPath(p.Root), "kernel_image")
```

Comment:
This rename here of the kernelPath file is just to avoid a confusing/inaccurate name.

Note that bzImage is the default make target for x86 builds, but that other architectures don't necessarily have a bzImage make target.  For this reason I removed bzImage as the target in the make invocation so it would build the default target which should generate a kernel image which will be usable for most other architectures.  KVM on arm64 for example needs an uncompressed vmlinux file, and vmlinux is generated by the arm64 default make target.